php/Terminus/Helpers/UpdateHelper.php

```diff
@@ -17,11 +17,13 @@ class UpdateHelper extends TerminusHelper
     public function getCurrentVersion()
     {
         $request  = new Request();
-        $url      = 'https://api.github.com/repos/pantheon-systems/terminus/releases';
-        $url     .= '?per_page=1';
-        $response = $request->request($url, ['absolute_url' => true]);
-        $release  = array_shift($response['data']);
-        $cache    = new FileCache();
+        $url = 'https://api.github.com/repos/pantheon-systems/terminus/releases/tags/0.x?per_page=1';
```

Comment:
@TeslaDethray I just did a quick check and this API URL always returns the single latest **non-pre-release** release:
`https://api.github.com/repos/pantheon-systems/terminus/releases/latest`
I think that means we can use the above URL for all versions as long as we mark future 1.0.0-alpha releases with the `pre-release` checkbox.